### ISSUE TITLE
Method `get_diff` not found

### ISSUE THREAD

**Issue by Diogo-Rossi** [deshaw/versioned-hdf5#320]
Hi.
Thank you for this great work in versioned hdf5 library. It looks really good!

I just tried to use a method found in the documentation ([`get_diff`](https://deshaw.github.io/versioned-hdf5/reference.html#versioned_hdf5.api.VersionedHDF5File.get_diff)) to compute the difference between two versions of a dataset, and got the following error:

```
In [9]: file.get_diff
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
Cell In[9], line 1
----> 1 file.get_diff

AttributeError: 'VersionedHDF5File' object has no attribute 'get_diff'
```

It seems that the method is public documented, but is not available yet in the current release. Is that right?

**Comment by peytondmurray** [comment]
@Diogo-Rossi You're absolutely right - the current documentation is for the `master` branch. We should track different versions of the documentation - I've made an issue to track this work: #328. I'm going to close this, and I'll track follow up work in that issue.
